_rules/first-focusable-elements-are-links-to-content-e53727.md

```diff
@@ -0,0 +1,960 @@
+		<nav>
+			<a href="#bio-translator">Skip to translator's biography</a>
+			<a href="#about-book">Skip to information about the book</a>
+			<span id="link-main" onclick="document.getElementById('main').focus()" tabindex="0">Skip to text</span>
+		</nav>
+
+		<aside id="bio-translator">
+			<h1>About the translator</h1>
+			<p>Yu Sumei is a professor of English at East China Normal University.</p>
+		</aside>
+		<aside id="about-book">
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+
+		<main id="main">
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+			<a href="/test-assets/bypass-blocks-cf77f2/chapter1.html">Read Chapter 2</a>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 8
+
+The first two [focusable][] elements form an [initial segment][] that fulfills Expectation 1 but not Expectation 2 because there is no link to the [main block of content][]. The third [focusable][] element does not fulfill Expectation 1 because it cannot be [activated][] by keyboard. Thus, no [initial segment][] fulfills both Expectations.
+
+```html
+<html lang="en">
+	<head>
+		<script src="../test-assets/bypass-blocks-cf77f2/click-on-enter.js"></script>
+		<title>The Three Kingdoms, Chapter 1</title>
+	</head>
+	<body onload="ClickOnEnter(['link-bio', 'link-about'])">
+		<nav>
+			<span role="link" onclick="location.href='#bio-translator';" tabindex="0" id="link-bio"
+				>Skip to translator's biography</span
+			>
+			<span role="link" onclick="location.href='#about-book';" tabindex="0" id="link-about"
+				>Skip to information about the book</span
+			>
+			<span role="link" onclick="location.href='#main';" tabindex="0" id="link-main">Skip to text</span>
+		</nav>
+
+		<aside id="bio-translator">
+			<h1>About the translator</h1>
+			<p>Yu Sumei is a professor of English at East China Normal University.</p>
+		</aside>
+		<aside id="about-book">
+			<h1>About the book</h1>
+			<p>The Romance of the Three Kingdoms is a 14th century historical novel.</p>
+		</aside>
+
+		<main id="main">
+			<h1>Three Heroes Swear Brotherhood at a Feast in the Peach Garden</h1>
+			<p>
+				Unity succeeds division and division follows unity. One is bound to be replaced by the other after a long span
+				of time.
+			</p>
+			<a href="/test-assets/bypass-blocks-cf77f2/chapter1.html">Read Chapter 2</a>
+		</main>
+	</body>
+</html>
+```
+
+#### Failed Example 9
+
+The first two [focusable][] elements form an [initial segment][] that fulfills Expectation 1 but not Expectation 2 because there is no link to the [main block of content][]. The third [focusable][] element does not fulfill Expectation 1 because its [accessible name][] does not communicate the intend. Thus, no [initial segment][] fulfills both Expectations.
```

Comment:
```suggestion
The first two [focusable][] elements form an [initial segment][] that fulfills Expectation 1 but not Expectation 2 because there is no link to the [main block of content][]. The third [focusable][] element does not fulfill Expectation 1 because its [accessible name][] does not communicate the intent. Thus, no [initial segment][] fulfills both Expectations.
```